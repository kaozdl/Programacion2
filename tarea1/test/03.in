10 > 5 * 2
foo # bar
otra linea
mas o menos
mitad
fin? 
despues llega
sdfsd gsdg
ahora casi termina
es el ultimo
11sera el resultado. El string '11sera' se convierte al natural 11.
