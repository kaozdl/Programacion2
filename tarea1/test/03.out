10 > 5 * 2|foo # bar|otra linea|mas o menos|mitad|fin? |despues llega|sdfsd gsdg|ahora casi termina|es el ultimo
Longitud  Texto 
      10  10 > 5 * 2
       9  foo # bar
      10  otra linea
      11  mas o menos
       5  mitad
       5  fin? 
      13  despues llega
      10  sdfsd gsdg
      18  ahora casi termina
      12  es el ultimo
mitad > 10 > 5 * 2
mitad > foo # bar
mitad < otra linea
mitad > mas o menos
mitad = mitad
mitad > fin? 
mitad > despues llega
mitad < sdfsd gsdg
mitad > ahora casi termina
mitad > es el ultimo
La conversión es 11.
El inicio numérico de la palabra leida era 11.
